force_wfmanager/wfmanager.py

```diff
@@ -65,8 +65,10 @@ def _remove_tasks(self):
                 window.remove_task(task)
 
     # FIXME: If the underlying envisage TasksApplication function is fixed to
-    #        work correctly, this will not be needed
+    #        work correctly, this will not be needed.
     def create_window(self, layout, restore, **traits):
+        """ Creates a new TaskWindow.
+        """
```

Comment:
(This is not strictly related to this PR)

This was required by a warning in Sphinx, treated as fatal: https://travis-ci.com/force-h2020/force-wfmanager/builds/115790867#L772

and it was probably caused by the colon after `Parameters` here:

https://github.com/enthought/envisage/blob/01696d52807747c8917d486e3852acf166ae0076/envisage/ui/tasks/tasks_application.py#L171-L175

Reply:
Do you know whether the FIXME statements referring to `traitsui/qt4/ui_panel.py` in `wfmanager.py` are addressed in the library upgrades?

Reply:
I haven't checked. Checking now.

Reply:
I'm still not sure what was happening that required #228 (which introduced these FIXMEs). However, we haven't got any functional updates in https://github.com/enthought/traitsui/commits/master/traitsui/qt4/ui_panel.py, and removing the overrides fails this test:

https://github.com/force-h2020/force-wfmanager/blob/b7db3c397369bd2cc497eb06f5269e0f44b0c1ca/force_wfmanager/tests/test_wfmanager.py#L68-L72

Reply:
Ok, cheers for checking. I can't seem to find an issue related to these FIXMEs - could you raise one referencing the library dependencies being updated so as to keep track of them in the future?

Reply:
Done: #300